dev/deps/spark-deps-hadoop-2.7-hive-2.3

```diff
@@ -35,7 +35,7 @@ commons-cli/1.2//commons-cli-1.2.jar
 commons-codec/1.10//commons-codec-1.10.jar
 commons-collections/3.2.2//commons-collections-3.2.2.jar
 commons-compiler/3.0.16//commons-compiler-3.0.16.jar
-commons-compress/1.8.1//commons-compress-1.8.1.jar
+commons-compress/1.4.1//commons-compress-1.4.1.jar
```

Comment:
Sorry, but is this downgraded?

Reply:
Good catch. I messed up dependencies in parent pom. Fixed.